niet/__main__.py

```diff
@@ -1,15 +1,95 @@
 #!/usr/bin/python
 import argparse
 import json
-import sys
 import os.path
+import sys
+from future.utils import viewitems
 import yaml
 
 
+try:
+    IFS = os.environ['IFS']
```

Comment:
instead of using a `try/except` behavior prefer use `IFS = os.getenv('IFS', ' ')` 
https://docs.python.org/2/library/os.html#os.getenv